packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHeadersHidden.4.doc.mdx

```diff
@@ -13,9 +13,9 @@ This is useful when needing to use custom alternative to column headers, such as
 **Note:**
 - Column header labels should still be provided in the dataset to ensure that sceenreaders will still read out the appropriate context for the table.
 - When using custom column headers, then logic must be added to ensure that the column widths match are in sync if resizing columns are enabled.
-The [columns.width](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#column) property for columns can be used to set the column widths programatically.
+The [width](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#column) property of columns can be used to control the Flowsheet Data Grid column widths.
```

Comment:
NIT: I like that there is a link to the width prop here. I noticed that in line 9 there is a mentioning of hasVisibleColumns prop. Might be consistent to make it a link as well.